src/phoenix/core/traces.py

```diff
@@ -36,17 +36,17 @@ def __init__(self) -> None:
 
     def get_project(self, project_name: str) -> Optional["Project"]:
         with self._lock:
-            return self._projects.get(project_name)
+            return self._projects.get(project_name, Project())
 
     def get_projects(self) -> Iterator[Tuple[int, str, "Project"]]:
         with self._lock:
             for project_id, (project_name, project) in enumerate(self._projects.items()):
                 if project.is_archived:
```

Comment:
can't you remove the archived notion now?

Reply:
yea, I could do that